src/models/calendar.ts

```diff
@@ -4,8 +4,26 @@ import { Types } from 'mongoose'
 // Sub Models.
 import Location from './subModels/shared/location'
 
+// Required local Namespace.
+import Event from './event'
+
 namespace Calendar {
 
+	// --------------------------------
+	// Server Responses
```

Comment:
I think we should put the server responses in a different Namespace so we can keep model definitions and route definitions separate. Also adding comment breaks (//--------------) that describe the type of server response. For example not all Calendar server responses are of the 'paged' response type.